miio/tests/test_devicestatus.py

```diff
@@ -140,6 +140,7 @@ def level(self) -> int:
             return 1
 
     mocker.patch("miio.Device.send")
+    mocker.patch("miio.Device.send_handshake")
```

Comment:
No it was needed because the tests otherwise fail, I can't exactly remember why but somehow the device_id is used somewhere, and when that is needed, the send_handshake was called.

Will test some more later tonight.

Reply:
You were right, the `mocker.patch("miio.Device.send_handshake")` is not needed and I have removed it.
I was missing one patch of the device_id.